src/pages/booklist/components/Textbook.tsx

```diff
@@ -49,77 +39,82 @@ export function Textbook({
     url && logEvent('textbook_click', { url });
   };
 
+  amazonUrl = amazonUrl || `https://amazon.ca/dp/${ISBN.toIsbn10(isbn)}`;
+
   return (
-    <Flex alignItems="center" direction={{ base: 'column', md: 'row' }} mt="1">
-      {/* TODO: Replace with image from API */}
-      {/* <Image src={textbook.textbooks[0].imageUrl} /> */}
-      <Center
-        h="167px"
-        w="120px"
-        bgColor={mode('rgba(114, 114, 114, 0.459)', 'rgba(58, 58, 58, 0.459)')}
-        mr="1"
-        borderRadius="5"
-      >
-        <IoBook size="3em" />
-      </Center>
-      <VStack ml="1" h="100%" alignItems={{ base: 'center', md: 'start' }}>
-        <HStack>
-          <Heading size="sm">{title}</Heading>
+    <Flex
+      key={`${isbn}`}
+      alignItems="center"
+      justifyContent="space-between"
+      direction={{ base: 'column', md: 'row' }}
+      gap="8"
+      boxShadow="md"
+      borderRadius="lg"
+      p="4"
+      pt={{ base: '8', md: '4' }}
+      backgroundColor={mode('gray.100', 'gray.700')}
```

Comment:
Can we use the colours defined [here](https://github.com/VikeLabs/courseup/blob/main/src/lib/theme.tsx#L14-L15)?
i.e. `light.background` and `dark.background`